api-reference/beta/api/device_post_devices.md

```diff
@@ -47,13 +47,7 @@ Content-length: 364
 
 {
```

Comment:
I thought the point was that we DON'T want folks creating new device objects through Microsoft Graph.  Either we mark this as deprecated OR we remove the topic and the ToC entirely.